drivers/cyw43/cywbt.c

```diff
@@ -149,10 +149,14 @@ STATIC int cywbt_download_firmware(const uint8_t *firmware) {
     }
 
     // RF switch must select high path during BT patch boot
+    #if MICROPY_HW_HAS_RF_SWITCH
```

Comment:
I'd rather call this `MICROPY_HW_ENABLE_RF_SWITCH`  (the board may have one but not want to enable it).